docs/example/vllm/README.md

```diff
@@ -0,0 +1,101 @@
+# Deploy Distributed Inference Service with vLLM and LWS on GPUs
+
+In this example, we will use LeaderWorkerSet to deploy a distributed inference service with vLLM on GPUs.    
+[vLLM](https://docs.vllm.ai/en/latest/index.html) supports distributed tensor-parallel inference and serving. Currently, it supports Megatron-LM’s tensor parallel algorithm. It manages the distributed runtime with [Ray](https://docs.ray.io/en/latest/index.html). See the doc [vLLM Distributed Inference and Serving](https://docs.vllm.ai/en/latest/serving/distributed_serving.html) for more details.
+
+## Install LeaderWorkerSet
+
+Follow the step-by-step guide on how to install LWS. [View installation guide](https://github.com/kubernetes-sigs/lws/blob/main/docs/setup/install.md)
+
+## Deploy LeaderWorkerSet of vLLM
+We use LeaderWorkerSet to deploy two vLLM model replicas, and each vLLM replica has two pods (tp=2). 
+The leader pod runs the Ray head and the http server, while the workers run the Ray workers.
+
+```shell
+kubectl apply -f lws.yaml
+```
+
+Verify the status of the vLLM pods
+```shell
+kubectl get pods
+```
+
+Should get an output similar to this
+```shell
+NAME       READY   STATUS    RESTARTS   AGE
+vllm-0     1/1     Running   0          2s
+vllm-0-1   1/1     Running   0          2s
+vllm-1     1/1     Running   0          2s
+vllm-1-1   1/1     Running   0          2s
+```
+
+Verify that the distributed tensor-parallel inference works
+```shell
+kubectl logs vllm-0 |grep -i "Loading model weights took" 
+```
+Should get an output similar to this
+```text
+INFO 05-08 03:20:24 model_runner.py:173] Loading model weights took 0.1189 GB
+(RayWorkerWrapper pid=169, ip=10.20.0.197) INFO 05-08 03:20:28 model_runner.py:173] Loading model weights took 0.1189 GB
+```
+
+
+# Use vLLM
```

Comment:
Can we remove this because it's clear enough and actually it's not recommended to add two first level # at markdown. Thanks.

Reply:
removed